lib/rucio/tests/test_did_meta_plugins.py

```diff
@@ -455,145 +455,6 @@ def test_get_metadata(self, mock_scope, did_client, db_session):
             assert all_metadata['key2'] == value2
             assert all_metadata['project'] == "data12_14TeV"
 
-    @pytest.mark.dirty
-    @pytest.mark.noparallel(reason='fails when run in parallel')
-    def test_list_dids_extended(self, did_client, db_session):
```

Comment:
Couldn't this test just be re-written to use `list_dids` instead of deleting it?

Reply:
yes it could, my mistake - I hadn't realised these were tests for the client